source/containers.tex

```diff
@@ -3125,12 +3181,18 @@
 
 \pnum
 The member function templates
-\tcode{find}, \tcode{count}, \tcode{equal_range}, and \tcode{contains}
+\tcode{find}, \tcode{count}, \tcode{equal_range}, \tcode{contains},
+\tcode{extract}, and \tcode{erase}
 shall not participate in overload resolution unless
 the \grammarterm{qualified-id}s
 \tcode{Pred::is_transparent} and
 \tcode{Hash::is_transparent}
 are both valid and denote types\iref{temp.deduct}.
+Additionally, the member function templates \tcode{extract} and \tcode{erase}
```

Comment:
"erase and extract" or "extract and erase"? Please be consistent (see comment above).